databricks/koalas/frame.py

```diff
@@ -331,8 +331,10 @@ def assign(self, **kwargs):
         for (name, c) in pairs:
             if isinstance(c, Series):
                 sdf = sdf.withColumn(name, c._scol)
-            else:
+            elif isinstance(c, Column):
```

Comment:
I guess we don't need this clause because `F.lit()` will handle `Column` properly.

Reply:
Yup. Let me actually just leave it as is - we will probably have to need this `else` to support other types like numpy anyway.

Reply:
sure, that's fine.